admin/class-yoast-notification-center.php

```diff
@@ -769,4 +761,40 @@ private function queue_transaction( $callback, $args ) {
 	private function add_transaction_to_queue( $callback, $args ) {
 		$this->queued_transactions[] = array( $callback, $args );
 	}
+
+	/**
+	 * Removes all notifications from storage.
+	 *
+	 * @return bool True when notifications got removed.
+	 */
+	protected function remove_storage() {
+		if ( ! $this->has_stored_notifications() ) {
+			return false;
+		}
+
+		delete_user_option( get_current_user_id(), self::STORAGE_KEY );
```

Comment:
Tip: you can return the value of `delete_user_option` directly ;)

Reply:
That will not work for the current test implementation.